src/fft_cufft.f90

```diff
@@ -11,6 +11,7 @@ module decomp_2d_fft
    use iso_c_binding
    use cudafor
    use cufft
+   use m_decomp_pool
```

Comment:
Since we do not use the memory pool for GPU yet, do we need this just for the code to compile? Does it compile without this module?

Reply:
I think compilation will fail because `#include "fft_common.f90"`

Reply:
I simply commented it and i managed to compile and run. This was for a CUMPI. I'll try also with NCCL

Reply:
Yes it seems to work. Do we keep it or not?